risc0/build/src/lib.rs

```diff
@@ -441,14 +447,18 @@ impl Default for GuestOptions {
 /// Specify custom options for a guest package by defining its [GuestOptions].
 /// See [embed_methods].
 pub fn embed_methods_with_options(mut guest_pkg_to_options: HashMap<&str, GuestOptions>) {
-    let skip_var_name = "RISC0_SKIP_BUILD";
-    println!("cargo:rerun-if-env-changed={}", skip_var_name);
-    if env::var(skip_var_name).is_ok() {
-        return;
-    }
-
     let out_dir_env = env::var_os("OUT_DIR").unwrap();
-    let out_dir = Path::new(&out_dir_env);
+    let out_dir = Path::new(&out_dir_env); // $ROOT/target/$profile/build/$crate/out
+    let guest_dir = out_dir
```

Comment:
Should we perform a sanity check here to ensure the resolved `guest_dir` is a cargo `target/$profile` directory? Just in case someone invokes risc0-build in a non-standard layout.

Reply:
I actually don't know what situation would cause it to be a different layout, and I'm not sure what sanity check I should perform, any suggestions?